### PR TITLE
Should fix raise-missing-from failure

### PR DESCRIPTION
**Pull request by dherrada**

### CODE REVIEW

**Comment by dherrada** [comment]
Not entirely sure if this is the right way to do it

**Comment by kattni** [comment]
@dherrada I was about to say there's another way to do it, but I'm conflating it with exceptions. Do you have documentation to support this solution? I've not dealt with it previously.;

**Comment by dherrada** [comment]
@kattni 
https://stackoverflow.com/a/63852262
Looks like I should add an `as e` to raise it from

File: adafruit_hue.py [hunk 1]
```diff
@@ -115,11 +115,11 @@ def discover_bridge(self):
             json_data = resp.json()
             bridge_ip = json_data[0]["internalipaddress"]
             resp.close()
-        except:
+        except Exception as e:
```
Comment (kattni) [inline adafruit_hue.py:118]: Should this be `except TypeError as e:`? I feel like that's what I've seen previous, but I could be mistaken.

Comment (dherrada) [reply adafruit_hue.py:118]: @kattni I'm not really sure

Comment (kattni) [reply adafruit_hue.py:118]: I'm good with it as it is then, as long as we can get it passing.

**Review by kattni — APPROVED** [review]

**Pull request merged**